CHANGELOG.md

```diff
@@ -11,6 +11,7 @@
 * [CHANGE] Memcached: Remove experimental `-<prefix>.memcached.addresses-provider` flag to use alternate DNS service discovery backends. The more reliable backend introduced in 2.16.0 (#10895) is now the default. As a result of this change, DNS-based cache service discovery no longer supports search domains. #12175
 * [CHANGE] Query-frontend: Remove the CLI flag `-query-frontend.downstream-url` and corresponding YAML configuration and the ability to use the query-frontend to proxy arbitrary Prometheus backends. #12191
 * [CHANGE] Query-frontend: Remove experimental instant query splitting feature. #12267
+* [CHANGE] Query-frontend: Remove the CLI flags `-querier.max-outstanding-requests-per-tenant` and `-query-frontend.querier-forget-delay` and corresponding YAML configuration. This is part of a change that makes the query-scheduler a required component. A result of this removes the ability to run the query-frontend with an embedded query-scheduler. Instead, a dedicated query-scheduler component must be run. #12200
```

Comment:
```suggestion
* [CHANGE] Query-frontend: Remove the CLI flags `-querier.max-outstanding-requests-per-tenant` and `-query-frontend.querier-forget-delay` and corresponding YAML configurations. This is part of a change that makes the query-scheduler a required component. This removes the ability to run the query-frontend with an embedded query-scheduler. Instead, you must run a dedicated query-scheduler component. #12200
```

Reply:
Fixed in fda0fda357884f4213f25572d2740087ecca1cb6